api/src/main/java/marquez/db/JobDao.java

```diff
@@ -386,4 +447,64 @@ JobRow upsertJob(
       String location,
       UUID symlinkTargetId,
       PGobject inputs);
+
+  @SqlUpdate(
+      """
+    WITH new_tag AS (
+    INSERT INTO tags (uuid, created_at, updated_at, name, description)
+    SELECT
+      gen_random_uuid(),
+      NOW(),
+      NOW(),
+      :tagName,
+      'No Description'
```

Comment:
Is there a reason we would want to hardcode this in the SQL, instead of in our application code to be a bit more flexible?

Reply:
Yeah I can leave as null - was just being explicit more than anything.